apps/site/src/app/(home)/page.tsx

```diff
@@ -1,7 +1,18 @@
 import { Landing } from "./sections";
+import FAQ from "./sections/FAQ";
+
+export const revalidate = 60;
 
 export default function Home() {
 	// Show landing section only if still in maintenance,
 	// otherwise show the rest of the sections
-	return process.env.MAINTENANCE_MODE_HOME ? <Landing /> : <Landing />;
+	return process.env.MAINTENANCE_MODE_HOME ? (
```

Comment:
Nitpick [repeating]: The width of your indents seems relatively large. Please make sure to set it to 4 spaces if you can.

Reply:
interesting, it looks fine on vscode and it is set to 4 spaces

Reply:
Might just be a GitHub thing then.